/gauge_detector:
  ros__parameters:
    qos_overrides:
      /gauge_image:
        publisher:
          reliability: best_effort
          durability: volatile
          history: keep_last
          depth: 1
      /image:
        subscription:
          reliability: best_effort
          durability: volatile
          history: keep_last
          depth: 1
/gauge_reader:
  ros__parameters:
    qos_overrides:
      /image:
        subscription:
          reliability: best_effort
          durability: volatile
          history: keep_last
          depth: 1
      /gauge_reading:
        publisher:
          reliability: reliable
          durability: volatile
          history: keep_last
          depth: 5
